provisioning/ansible/playbooks/www.yml

```diff
@@ -28,6 +28,9 @@
       - {
         servername: "favrskov.loc",
         serveralias: "*.*",
+        serveralias1: "designguiden.favrskov.loc",
```

Comment:
Is that used properly in Apache configuration?

Reply:
yep